src/store/reducers/socialLoginReducer.js

```diff
@@ -0,0 +1,28 @@
+import { SOCIAL_LOGIN_SUCCESS, SOCIAL_LOGIN_FAILURE } from '../actions/actionTypes';
+
+
+const initialState = {
+  logged_in: false,
+  errors: '',
+};
+
+const socialLoginReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SOCIAL_LOGIN_SUCCESS:
+      return {
+        ...state,
+        logged_in: true,
+        errors: '',
+      };
+    case SOCIAL_LOGIN_FAILURE:
+      return {
+        ...state,
+        logged_in: false,
+        errors: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
+export default socialLoginReducer;
```

Comment:
@cob04, you do not need a different reducer for this, the whole application can use one login reducer. Kindly use @MandelaK's reducer or create one that is not specific to OAuth.

Reply:
I have switched to using `loginReducer`.